src/pages/basic_data/institution_admin/edit_user/UserFormDialog.tsx

```diff
@@ -38,24 +51,36 @@ export interface UserFormData {
   user?: InstitutionUser;
 }
 
-export const UserFormDialog = ({ open, onClose, existingPerson }: UserFormDialogProps) => {
+export const UserFormDialog = ({ open, onClose, existingUser, existingPerson }: UserFormDialogProps) => {
   const { t } = useTranslation();
   const dispatch = useDispatch();
   const user = useSelector((store: RootState) => store.user);
   const topOrgCristinId = user?.topOrgCristinId;
   const customerId = user?.customerId ?? '';
 
-  const personCristinIdentifier = getIdentifierFromId(existingPerson.id);
+  const personId = typeof existingPerson === 'string' ? existingPerson : '';
```

Comment:
rart å defaulte til tom streng når vi har en id. selv om det ikke har noen praktisk betydning
```suggestion
  const personId = typeof existingPerson === 'string' ? existingPerson : existingPerson.id;
```